nessie/utils/constants.py

```diff
@@ -1,6 +1,15 @@
-#Urls
+# Urls
 baseUrl = "http://api.reimaginebanking.com"
+customersUrl = baseUrl + "/customers"
+accountsUrl = baseUrl + "/accounts"
+customersIdUrl = customersUrl + "/%s"
+accountsCustomerIdUrl = accountsUrl + "/%s/customer"
 
-#Exception Messages
-missingFields = 1
-invalidFields = 2
\ No newline at end of file
+# Exception Messages
+success = 0
+atmMissingFields = 1
+atmInvalidFields = 2
```

Comment:
Might want to use some sort of ENUM for this in the future. but we can leave it as is for now.